doc/source/release-notes/Percona-Server-5.6.43-84.3.rst

```diff
@@ -0,0 +1,56 @@
+.. rn:: 5.6.43-84.3
+
+================================================================================
+|Percona Server| |release|
+================================================================================
+
+Percona is glad to announce the release of |Percona Server| |release| on |date|
+(Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.6/Percona-Server-5.6.43-84.3/>`_
+and from the :doc:`Percona Software Repositories </installation>`).
+
+This release merges changes of `MySQL 5.6.43
+<http://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-43.html>`_, including
+all the bug fixes in it. |Percona Server| |release| is now the current GA
+release in the 5.6 series. All of Percona’s software is open-source and free.
+
+Bugs Fixed
+================================================================================
+
+- A sequence of LOCK TABLES FOR BACKUP and STOP SLAVE SQL_THREAD could cause
+  replication to be blocked and not possible to be restarted normally. Bug fixed
+  :psbug:`4758` (upstream :mysqlbug:`93649`).
+- In some cases, it was possible to inject SQL into the binary log due to
+  non-quoting in ROLLBACK TO SAVEPOINT. Bug fixed :psbug:`4791` (upstream
+  :mysqlbug:`92227`).
+- **http** with **https** in http://bugs.percona.com in server crash
+  messages. Bug fixed :psbug:`4855`.
+- Inconsistent ResultSet could be obtained for different execution plans. Bug
```

Comment:
Please use standard MySQL terms, not "ResultSet". Description too vague, please check with @percona-ysorokin for a more exact description